src/pages/Landing/index.tsx

```diff
@@ -306,9 +326,15 @@ export default function Landing() {
               name={SharedEventName.ELEMENT_CLICKED}
               element={InterfaceElementName.LANDING_PAGE_SWAP_ELEMENT}
             >
-              <Link to="/swap">
-                <LandingSwap />
-              </Link>
+              {swapWidgetEnabled ? (
```

Comment:
nit: You can inline this: `{useSwapWidgetEnabled() ? (`.

Reply:
i tried that but we can't because it would be conditionally called